stream_alert_cli/test.py

```diff
@@ -53,6 +54,45 @@ def report_output(cols, failed):
 
     print '\t{}\ttest ({}): {}'.format(status, *cols)
 
+def report_output_summary(rules_fail_pass):
+    """Helper function to print the summary results of all tests
+    Args:
+        rules_fail_pass [list]: A list containing two lists for failed and passed
+            rule tests. The sublists contain tuples made up of: (rule_name, rule_description)
+    """
+    failed_tests = len(rules_fail_pass[0])
+    passed_tests = len(rules_fail_pass[1])
+    total_tests = failed_tests + passed_tests
+
+    # Print a message indicating how many of the total tests passed
+    print '{}({}/{})\tTests Passed{}'.format(COLOR_GREEN, passed_tests, total_tests, COLOR_RESET)
+
+    # Check if there were failed tests and report on them appropriately
+    if rules_fail_pass[0]:
+        color = COLOR_RED
+        # Print a message indicating how many of the total tests failed
+        print '{}({}/{})\tTests Failed'.format(color, failed_tests, total_tests)
+
+        # Iterate over the rule_name values in the failed list and report on them
+        for i, failure in enumerate(rules_fail_pass[0]):
```

Comment:
nit: please use `indx` or equivalent instead of `i` (for here and the block below)